js/file.js

```diff
@@ -0,0 +1,18 @@
+function showCloseMenu() {
+  const menuSection = document.querySelector('#mobile-nav');
+  const nav = document.querySelector('#desktop-nav');
+  const headerName = document.querySelector('#logo');
+  const itemsContainer = document.querySelector('#items-container-menu');
+
+  menuSection.classList.toggle('d-none');
+  nav.classList.toggle('d-none');
+  headerName.classList.toggle('d-none');
+  headerName.classList.toggle('d-flex-center');
+  itemsContainer.classList.toggle('d-none');
+  itemsContainer.classList.toggle('d-flex');
+}
+
+const hamburgerBtn = document.querySelector('.menu-bars');
+const closeBtn = document.querySelector('#header-close-menu');
+hamburgerBtn.addEventListener('click', showCloseMenu);
+closeBtn.addEventListener('click', showCloseMenu);
```

Comment:
- [ ] Please check your functions that open and close the mobile menu. Right now, the menu is not being shown, and obviously, if you cannot open the menu, you cannot close it as well.

Reply:
- [ ] Also, kindly try to add a good README.md file. It is important to document your projects, and a README file it's an important part of the documentation, so please put your hands on it! 🙌